1209/9317/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: Palawan-Tx
+owner: Kosagi
+license: CC4.0-BY-SA (hardware) / MUT (firmware)
```

Comment:
I've not got direct write access (although @Arachnid did offer), I'd either do a PR or see if he offers to edit it in-place.